tico/serialize/circle_serializer.py

```diff
@@ -47,7 +48,7 @@ def _initialize_model() -> tuple[CircleModel, CircleSubgraph]:
     return model, graph
 
 
-def build_circle(ep: ExportedProgram) -> bytes:
+def build_circle(ep: ExportedProgram, config: CompileConfigBase) -> bytes:
```

Comment:
Setting default value keeps current behavior of `build_circle`. I think `test/unit_test/quantization_test/test_circle_executor.py` was modified because of this.

```suggestion
def build_circle(ep: ExportedProgram, config: CompileConfigBase = get_default_config()) -> bytes:
```